package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "build": "node build",
-    "test": "node verify"
+    "test": "node verify",
+    "prepush": "npm run test",
+    "postinstall": "if [ -d .git-hooks ]; then for file in `ls .git-hooks`; do ln -sf ../../.git-hooks/${file} .git/hooks/${file}; done fi"
```

Comment:
Actually, we are checking for the `.git-hooks` directory just to find out if we should install the customized hooks. Since this directory is ignored when the library is installed, we won't be installing and executing our hooks in the consumer projects. 

As for the `.git/hooks` I have assumed that the project is always initialized as `.git` repository, do you think it makes sense to change this behaviour?